UI/window-basic-main.cpp

```diff
@@ -2018,20 +2018,16 @@ void OBSBasic::OBSInit()
 
 	ui->sources->UpdateIcons();
 
-#if !defined(_WIN32)
+#if !defined(_WIN32) && !defined(__APPLE__)
 	delete ui->actionShowCrashLogs;
 	delete ui->actionUploadLastCrashLog;
 	delete ui->menuCrashLogs;
+	delete ui->actionCheckForUpdates;
 	ui->actionShowCrashLogs = nullptr;
 	ui->actionUploadLastCrashLog = nullptr;
 	ui->menuCrashLogs = nullptr;
-#if !defined(__APPLE__)
-	delete ui->actionCheckForUpdates;
 	ui->actionCheckForUpdates = nullptr;
-#endif
-#endif
-
-#if defined(_WIN32) || defined(__APPLE__)
+#elif _WIN32 || __APPLE__
```

Comment:
This should be using `defined` instead of testing the values directly.

Reply:
This is some of the code that only got reverted from the commit removing the buttons.
Should I still clean it up?

Reply:
I'd say yes, purely as we (incorrectly) use a mix of both methods.

Reply:
Fixed.